README.md

```diff
@@ -21,3 +21,28 @@ A multiarch [tang] Docker image, based on [Alpine Linux].
 
 [Alpine Linux]: https://alpinelinux.org/
 [tang]:         https://github.com/latchset/tang
+
+
+# Quick-start
+To start a one-shot container simply run
+`docker run -it --rm -p 8080:8080/tcp padhihomelab/tang`
+where the standard server port, `8080`, will be exposed on your host machine.
+
+## Persist data
+The "quick-start command" will not persist its data (located at `/data`) beyond the lifetime of the container.  
+To link the tang server's data with a persistent volume on the host run:  
+`docker run -it --rm -p 8080:8080 --volume "/home/myUser/whatever":"/data" padhihomelab/tang`  
```

Comment:
Note that the files written to the host directory may not be owned by the host user running the container. They would be typically owned by the UID `12345` (the default UID for the `user` in the container).

The fix involves adding a `-e` parameter that modifies the UID to associate with `user` in the container:
`docker run -e DOCKER_UID=$(id -u) -it --rm -p 8080:8080 --volume "/home/myUser/whatever":"/data" padhihomelab/tang`

You could also add a note for interested users to find more details at: https://github.com/padhi-homelab/docker_alpine-base/#do-i-ever-need-to-override-the-uid-gid-etc-why-are-these-variables-exposed